modules/cnv-uploading-local-disk-image-dv.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-uploading-local-disk-images-block.adoc
+
+[id="cnv-uploading-local-disk-image-dv_{context}"]
+= Uploading a local disk image to a new DataVolume
+
+You can use the `virtctl` CLI utility to upload a virtual machine disk image from
+a client machine to a DataVolume (DV) in your cluster. Once uploaded, the DV can be 
+added to a virtual machine.
+
+.Prerequisites
+
+* A virtual machine disk image, in RAW, ISO, or QCOW2 format, optionally
```

Comment:
Please be consistent with periods/no periods at the end of your prereqs lists.

Reply:
I've fixed (added) the files relevant to this PR, but there are other inconsistencies in other modules. Once I'm done with this I'll open a new PR to make them consistent in all CNV docs.